TackleTabby/Assets/Scripts/Audio/AudioHandler.cs

```diff
@@ -25,4 +40,26 @@ public void ChangeMusicParameter(int ParameterValue)
     {
         _backgroundMusic.setParameterByName("Catalogue Open", ParameterValue);
     }
+
+    public void SetMusicVolume(float Volume)
```

Comment:
this doesn't follow our namingconvention
```suggestion
    public void SetMusicVolume(float volume)
```